components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/SessionSuggestionProvider.kt

```diff
@@ -26,6 +27,7 @@ class SessionSuggestionProvider(
     private val store: BrowserStore,
     private val selectTabUseCase: TabsUseCases.SelectTabUseCase,
     private val icons: BrowserIcons? = null,
+    private val indicatorIcon: Drawable? = null,
```

Comment:
`AwesomeBarFeature` has methods like `addSessionProvider()`. Can you add this icon to that method too? It is not used by Fenix but others like Reference Browser.

Reply:
Yep got it but in `AwesomeBarFeature` there is nothing like `SessionSuggestionProvider` or `BookmarkSuggestionProvider`. So it's unnecessary?

Reply:
`AwesomeBarFeature.addSessionProvider()`?

Reply:
https://github.com/mozilla-mobile/android-components/blob/a5509b6ab4a9db4420cd0f47d2e26f611e86f991/components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/AwesomeBarFeature.kt#L89

For example this one using SearchSuggestionProvider and in this file none of them using SessionSuggestionProvider and BookmarkSuggestionProvider. We're pushing icons to just SessionSuggestionProvider  and BookmarkSuggestionProvider.

Reply:
Ohhh, missed it. Okey.

Reply:
But there isn't bookmarkprovider, right?